joomla/joomla-cms
# Emptystate layout is used for non-empty cache

### Summary of Changes

"You have no cached content." is displayed after deleting the filtered cache group.

### Testing Instructions

1. Ensure that Joomla caching is enabled, load frontend as guest
2. Go to Maintenance > Clear Cache in backend
3. Filter by any frontend-specific cache group, i.e. `com_content` in search field.
4. See single group filtered
5. Delete it
6. See "You have no cached content." layout without search field.

### Actual result BEFORE applying this Pull Request

Frustrating "You have no cached content." message on active search filter applied though we have non-filtered cache groups.
Search field is not visible.

### Expected result AFTER applying this Pull Request

See the usual empty table with "No Matching Results" message and visible search input.

### Documentation Changes Required

No.
